roadmap.md

```diff
@@ -0,0 +1,53 @@
+
+# Long term project milestones
+
+1) Identify soil carbon data sets of interest to community
+2) Harmonization scripts targeting the identified data sets
+3) Educational material for soil science community regarding data management and best practices
+
+Versions (with DOI) will be established at major content development milestones that can either include a significant number of datasets harmonized (10+) or refinement of other content.
+
+We always need:
+1) Copy editing for code and documentation
+2) Code review
+
+## Current TODO
+
+1) QA/QC scripts that identify likely duplicate values and flag likely outliers for different variables
```

Comment:
Outlier testing at this stage should only be based on information that can't be inferred numerically, e.g. the PI has said some particular group of measurements is wonky. Any actual exclusion shouldn't happen until analysis stage.

Reply:
I'm going to push back a little on this. I think we can do some basic QA/QC flagging here. For example, there is a pH value that is 8000+ in the ISCN3 database; I think we can safely flag that as unrealistic. I don't think we should filter that data point out but I'm ok with flagging it for closer examination.

Reply:
Oh agreed, that's fine. Just no data removal IMO.

Reply:
"can't be inferred numerically" - that was too strong.